backends-common/rabbitmq/src/main/java/org/apache/james/backends/rabbitmq/RabbitMQHealthCheck.java

```diff
@@ -48,7 +46,7 @@ public ComponentName componentName() {
     @Override
     public Result check() {
         try {
-            if (connectionPool.tryConnection() && rabbitChannelPoolImpl.tryChannel()) {
```

Comment:
We don't try acquiring a channel anymore?

Reply:
I'm not sure it still make sense